### ISSUE TITLE
Untabbing not working

### ISSUE THREAD

**Issue by qazip** [NeoVintageous/NeoVintageous#309]
### Description
Let's say you've the following text:

`
     string 1
     string 2
     blabla
`

If you do visual mode, select all the text and then click on "<", you expect the white space to dissapear (it's what happens in vim). However, I'm not getting that in ST3 with neovintageous. I get:

`
<  string 1
    string 2
>  blabla
`

Any idea why? Is it only on my end?

**Comment by gerardroche** [comment]
I don't seem to be able to reproduce it, unless I'm misunderstanding what you mean by expecting the whitespace to disappear.


```
    str|ing1

    string2
```

Where the cursor is `|`, then press `Vjj<`, should result in:

```
|string1

string2
```

Check you `.vintageourc` file to see if you have any mappings overrideing the `>` key.

What OS and version of NeoVintageous are you using?

Additional information, configuration or data that might be necessary to reproduce the issue. You can get debugging information by copy and pasting the output of the console `Menu > View > Show Console`.

Command and input logging can be enabled from the console (run the commands in input box at the bottom of the console): `sublime.log_commands(True)` for command logging, and `sublime.log_input(True)` input logging. See the [debugging guide](https://github.com/NeoVintageous/NeoVintageous/CONTRIBUTING.md#debugging) for more detailed debugging information.

**Comment by qazip** [comment]
@gerardroche, you're right. It works as expected until you open a .tex file. Please try with a .tex file and let me know how it works. It might be a latextools problem?

**Comment by gerardroche** [comment]
No, I can't reproduce it. 

Perhaps it's a specific piece of text causing the problem? 

Or a maybe a plugin is causing the issue, what plugins have you installed? 

To check that NeoVintageous is receiving key events in latext related syntaxes: 

`Menu > View > Show Console`

Enable logging: `sublime.log_input(True); sublime.log_commands(True)`

```
...
plugins loaded
>>> sublime.log_input(True); sublime.log_commands(True)
chr evt: V (0x56)
command: press_key {"key": "V"}
chr evt: j (0x6a)
command: press_key {"key": "j"}
chr evt: j (0x6a)
command: press_key {"key": "j"}
chr evt: < (0x3c)
command: press_key {"key": "<lt>"}
```

**Comment by qazip** [comment]
There seems to be a snippet of some sort:

```
command: press_key {"key": "V"}
key evt: down
command: press_key {"key": "<down>"}
key evt: down
command: press_key {"key": "<down>"}
chr evt: < (0x3c)
command: insert_snippet {"contents": "<${0:$SELECTION}>"}
key evt: escape
```

**Comment by gerardroche** [comment]
The reason this is happening is because LatexTools is overriding a bunch of keybindings. `>` is not the only key that LatexTools overrides, it overrides a lot, including `=`, `<`, `l`, '$`, `[`, `]`, etc. See https://github.com/SublimeText/LaTeXTools/blob/master/Default%20(Linux).sublime-keymap. 
So, in many contexts, in latex related files, NeoVintageous won't work. 

LatexTools unpacks itself on installation which means it gets loaded after NeoVintageous, so any LatexTools keybindings override NeoVintageous. If NeoVintageous unpacked itself on install it would override the latextools keybindings, because packages are loaded alphabetically. Of course, then the LatextTools keybindings wouldn't work.

I'm not sure there are any easy ways solve this issue. It's the nature of the LatexTools plugin. It's intentionally override those keys to add features for latex files.

If you prefer NeoVintageous keybindings over the LatextTools, install NeoVintageous manually. Installing manually installs the package in the directory where packages are unpacked, so as I described earlier, NeoVintageous will override the LatextTools keybindings because it will load after the LatextTools package.

**Comment by gerardroche** [comment]
A kindof hacky fix is to add this in you keybindings:

`Menu > Preferences > Key Bindings`

```
    {"keys": [">"],         "command": "_nv_feed_key", "args": {"key": ">"},       "context": [{"key": "vi_command_mode_aware"}]},
``` 

The reason this will work is because user key bindings are loaded last, so this should override the LatextTools keybinding.  This keybinding is taken directly from the NeoVintageous keybindings.

Note: The NeoVintageous keybindings are subject to change. So if it's changed in the future, you'll need to update your custom fix.
